src/remote_server.js

```diff
@@ -212,8 +212,10 @@ class RemoteServer {
 
     getPort().then(port => {
 
+      let path = path.resolve(__dirname, '../dist/bin/phantesta-server.js');
```

Comment:
This seems to be incorrect. If you have references to `dist` in this code, you won't ever be able to run that code in development (because the `dist` directory might not exist). I think it should be `path.resolve(__dirname, 'bin/phantesta-server.js')`. That will resolve to `src/bin/phantesta-server.js` when you're in development, and `dist/bin/phantesta-server.js` when you're in production.